jsk_panda_robot/README.md

```diff
@@ -0,0 +1,124 @@
+   * If your Ubuntu version is not supported, check JSK backup: https://drive.google.com/drive/folders/1FiQ4m3XtoDlwdRIq3H7LJv8T882SBVBl
+
+2. Install ROS packages:
+   ```bash
+   mkdir -p ~/franka_ws/src
+   cd ~/franka_ws/src
+   wstool init
+   wstool merge https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_panda_robot/jsk_panda_user.rosinstall
+   wstool update
+   cd ../
+   source /opt/ros/melodic/setup.bash
+   rosdep install -y -r --from-paths src --ignore-src --skip-keys=librealsense2,realsense2_camera  # See comments in jsk_panda.rosinstall
+   catkin build jsk_panda_startup jsk_panda_teleop
+   source devel/setup.bash
+   ```
+3. If you want to use TouchUSBs (haptic devices), see [this README](https://github.com/pazeshun/Geomagic_Touch_ROS_Drivers/tree/dual-phantom-readme#use-multiple-devices) to setup devices.
+   * Note that this README was written for old Ubuntu.
+     - On Ubuntu 18.04, run `Touch_Setup` instead of `/opt/geomagic_touch_device_driver/Geomagic_Touch_Setup`. If you want to check device status, run `Touch_Diagnostic` instead of `/opt/geomagic_touch_device_driver/Geomagic_Touch_Diagnostic`.
+
+### Installation for Panda Controller PC
+1. Please see and follow installation written in: https://frankaemika.github.io/docs/installation_linux.html
+   * Note that you need to install real-time kernel (`PREEMPT-PR` kernel) for real-time control.
+   * Ref: Current controller PC uses following kernel:  `dual-panda1`: `5.4.19-rt11`, `dual-panda2`: `5.4.93-rt51`
+2. Install librealsense2:
+   https://github.com/IntelRealSense/librealsense/blob/master/doc/distribution_linux.md#installing-the-packages
+3. Install OpenHaptics and Touch Device Driver from here: https://support.3dsystems.com/s/article/OpenHaptics-for-Linux-Developer-Edition-v34?language=en_US
+   * If your Ubuntu version is not supported, check JSK backup: https://drive.google.com/drive/folders/1FiQ4m3XtoDlwdRIq3H7LJv8T882SBVBl
+4. Install ROS packages:
+   ```bash
+   mkdir -p ~/franka_ws/src
+   cd ~/franka_ws/src
+   wstool init
+   wstool merge https://raw.githubusercontent.com/jsk-ros-pkg/jsk_robot/master/jsk_panda_robot/jsk_panda.rosinstall
+   wstool update
+   cd ../
+   source /opt/ros/melodic/setup.bash
+   rosdep install -y -r --from-paths src --ignore-src --skip-keys=librealsense2,realsense2_camera  # See comments in jsk_panda.rosinstall
+   catkin build jsk_panda_startup jsk_panda_teleop
+   source devel/setup.bash
+   ```
+
+
+## Running Dual-Panda
+### Boot robot
+1. Please turn on the controller box and unlock joints by accessing desk.
+### Via roseus
+1. Start controller on controller PC:
+   ```bash
+   ssh leus@dual-panda1.jsk.imi.i.u-tokyo.ac.jp  # Or ssh leus@dual-panda2.jsk.imi.i.u-tokyo.ac.jp
+   roslaunch jsk_panda_startup dual_panda1.launch  # Or roslaunch jsk_panda_startup dual_panda2.launch
```

Comment:
please add explanation on which one is dual_panda1 and dual_panda2

Reply:
Added